src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogSenderBuildListener.java

```diff
@@ -49,7 +39,7 @@ public OtelLogSenderBuildListener(BuildInfo buildInfo, Map<String, String> conte
     public synchronized final PrintStream getLogger() {
         if (logger == null) {
             try {
-                logger = new PrintStream(new PlainTextConsoleOutputStream(new OtelLogOutputStream(buildInfo, context)), false, "UTF-8");
```

Comment:
Why did you remove `PlainTextConsoleOutputStream`? To favor readability in Jenkins screens? what about the readability in the Otel Log storage?

Reply:
if we do not do it we can not restore the Pipeline headers on the retrieve, we do not want the Jenkins UI lost functionality if you use the plugin.

Reply:
can we have good readability in the observability backend if we keep Jenkins annotations?

Reply:
we retrieve the log files in pages, we have to make some stress tests to find the limits of the retrieve from Jenkins, I am concerned about retrieving the full text console output on jobs with a  huge log (GB)